<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditVariableDialog</class>
 <widget class="QDialog" name="EditVariableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Variable</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Value:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="typeCombo"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="valueEdit"/>
          </item>
          <item>
           <widget class="glabels::ColorButton" name="colorValueButton">
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="incrementGroup">
     <property name="title">
      <string>Increment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="incrementCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="stepSizeLabel">
          <property name="text">
           <string>Step size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stepSizeEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="paddingGroup">
     <property name="title">
      <string>Padding</string>
     </property>
     <layout class="QGridLayout" name="layoutPaddingGroup">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="padCheckBox">
          <property name="text">
           <string>Pad</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="padWidthLabel">
          <property name="text">
           <string>Pad Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="padWidthEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glabels::ColorButton</class>
   <extends>QPushButton</extends>
   <header>ColorButton.h</header>
   <slots>
    <signal>colorChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onTypeComboChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>incrementCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onIncrementComboChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stepSizeEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onStepSizeEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onNameEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onValueEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorValueButton</sender>
   <signal>colorChanged()</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onColorValueButtonChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>padCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onPadCheckBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>padWidthEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditVariableDialog</receiver>
   <slot>onPadWidthEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTypeComboChanged()</slot>
  <slot>onValueEditChanged()</slot>
  <slot>onIncrementComboChanged()</slot>
  <slot>onStepSizeEditChanged()</slot>
  <slot>onNameEditChanged()</slot>
  <slot>onColorValueButtonChanged()</slot>
  <slot>onPadCheckBoxChanged()</slot>
  <slot>onPadWidthEditChanged()</slot>
 </slots>
</ui>
